src/cli/core/utils/sort.js

```diff
@@ -8,8 +8,8 @@
  * @return {Int}   1 | 0 | -1
  */
 export function byDateDesc(a, b) {
-  var dateA = new Date(a.date)
-  var dateB = new Date(b.date)
+  var dateA = (a.abe_meta.publish != null && a.abe_meta.publish.latest != null) ? new Date(a.abe_meta.publish.latest.date) : 0
```

Comment:
You can't put this Abe logic into coreUtils.sort function. You should call the sort with the date you want to sort.